### PR TITLE
bugfix: adjust cluster-info cm networking

### PR DESCRIPTION
**Pull request by angao**
(cherry picked from commit 2ed431b0878d197612dc81dbb3b1eb4a263054f2)

**What type of PR is this?**
/kind bug
/area network

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: roles/self-hosted/templates/cluster-info-cm.yml.j2 [hunk 1]
```diff
@@ -9,7 +9,7 @@ data:
 {% if cluster_vip %}
   clusterVIP: "{{ cluster_vip }}"
 {% endif %}
-  networking: "{{ kube_network_plugin }}"
+  networking: "{{ (kube_network_plugin == 'bd-calico') | ternary('calico', kube_network_plugin) }}"
```
Comment (hanxueluo) [inline roles/self-hosted/templates/cluster-info-cm.yml.j2]: 有没有方法。判断 bd- 开头，就 trim 掉的？

Comment (angao) [reply roles/self-hosted/templates/cluster-info-cm.yml.j2]: 需要这样？感觉没必要？

Comment (hanxueluo) [reply roles/self-hosted/templates/cluster-info-cm.yml.j2]: 我觉得有必要。不然以后再使用个 calico 之外的 cni，这里就很不好写了。

Comment (angao) [reply roles/self-hosted/templates/cluster-info-cm.yml.j2]: 修改了下，看下

**Pull request merged**